Code/Framework/AzToolsFramework/Platform/Common/Default/AzToolsFramework/API/PythonLoader_Default.cpp

```diff
@@ -0,0 +1,14 @@
+/*
+* Copyright (c) Contributors to the Open 3D Engine Project.
+* For complete copyright and license terms please see the LICENSE at the root of this distribution.
+*
+* SPDX-License-Identifier: Apache-2.0 OR MIT
+*
+*/
+
+#include <AzToolsFramework/API/EditorPythonConsoleBus.h>
+
+namespace AzToolsFramework::EmbeddedPython
+{
+    void LoadPythonLib() {}
+}
```

Comment:
Instead of having a LoadPythonLib class, a default implementation for the PythonLoader class can be added here

```suggestion
namespace AzToolsFramework::EmbeddedPython
{
    PythonLoader::PythonLoader()
    {
    }
    
    PythonLoader::~PythonLoader()
    {
    }
}
```

Reply:
Working on it...